pennylane_rigetti/qc.py

```diff
@@ -193,7 +187,7 @@ def apply(self, operations, **kwargs):
         rotations = kwargs.get("rotations", [])
         self.prog += self.apply_rotations(rotations)
 
-        qubits = sorted(self.wiring.values())
+        qubits = self.wiring.values()
```

Comment:
```suggestion
        qubits = self._qubits
```
if we get rid of `self.wiring`